api/v1alpha1/search_types.go

```diff
@@ -85,13 +85,15 @@ type SearchDeployments struct {
 
 type DeploymentConfig struct {
 	// +optional
+	// +kubebuilder:validation:Minimum:=0
```

Comment:
should this be `Minimum:=1` instead of 0?

Reply:
Yes - this could go 1 . 0 does not add any value.